src/main/scala/nl/knaw/dans/easy/stage/fileitem/EasyFileMetadata.scala

```diff
@@ -15,13 +15,18 @@
  */
 package nl.knaw.dans.easy.stage.fileitem
 
+import java.io.File
+
 import scala.util.Try
 
 object EasyFileMetadata {
   def apply(s: FileItemSettings): Try[String] = Try {
+      val parentPath = s.pathInDataset.get.getParentFile
```

Comment:
You will get an error here (and in the next line) if `s.pathInDataset` is `None`. I assume that is why you have the `Try` wrapped around this method? Consider using `map` here to stay in the `Option` and (if necessary) convert to `Try` later.